### PR TITLE
refactor: remove medium and low prio code smells

### PR DESCRIPTION
**Pull request by nayyara-airlangga**

### CODE REVIEW

File: bot/repository.py [hunk 1]
```diff
@@ -55,32 +55,31 @@ def __init__(self, session: sessionmaker) -> None:
 
     def find_bots(self, skip: int, limit: int) -> list[Bot]:
         with self.create_session() as session:
-            return session.query(BotModel) \
-                .offset(skip) \
-                .limit(limit) \
-                .all() \
-                
+            return session.query(BotModel).offset(skip).limit(limit).all()
+
```
[the remark this answers is not in the document]
Comment (nayyara-airlangga) [reply bot/repository.py:59]: The change was made by a standard Python formatter called [Black](https://github.com/psf/black). I had it installed in my IDE so the code was auto-formatted when I make a change to a file using the IDE

[the remark this answers is not in the document]
Comment (Veivel) [reply bot/repository.py:59]: I see, do you think everyone else should set up Black as well in that case?

[the remark this answers is not in the document]
Comment (nayyara-airlangga) [reply bot/repository.py:59]: I don't think it's a hard requirement. But I do encourage the use of a standard formatter when writing code so we can focus on solving the problems and let the tool handle the tidyness

[the remark this answers is not in the document]
Comment (Veivel) [reply bot/repository.py:59]: Ok cool! Thanks for clarifying. LGTM

**Review by Veivel — APPROVED** [review]

**Pull request merged**